### DISCUSSION TITLE
Switching between editors issue

### DISCUSSION THREAD

**Discussion by teoxoy** [standardnotes/forum#2462] [General] [answered]
Using a dark theme like Futura and switching between notes that have different editors makes the editor area flash with a white background for a short period of time.

Steps to reproduce:
1. Install and activate Futura
2. Create a note that has the code editor as its editor
3. Create a 2nd note that has the bold editor as its editor
4. Switch between those and see the effect described above

I also tested the web app and the issue is not there, only happens on the windows app.

**Comment by bryvin** [comment]
This is also a side effect of caching; the desktop app recently underwent a change to how local extensions are served and they appear to not being cached so it's pulling off the local server each time you request an editor. This should be something we can make improvements on, thanks for letting us know.

**Comment by arielsvg** [comment]
This is affecting the web app as well; it happens because the editor is loaded in an iframe with a default white background. We'll have to investigate a way to keep it dark when using a dark theme 👍

**Comment by rcadariu** [comment]
+1 for this behaviour to be fixed.

**Comment by myreli** [comment, marked as answer]
Closing in favour of https://github.com/standardnotes/forum/discussions/2464, which reports the same problem and has more recent activity.
